.gitignore

```diff
@@ -10,6 +10,8 @@ parser_table.py
 parser_test_table.py
 xonsh/lexer_table.py
 xonsh/parser_table.py
+xonsh/completion_parser_table.py
+xonsh/parsers/completion_parser_table.py
 tests/lexer_table.py
```

Comment:
is it possible to fix the generated parser table to single location ? I too get this issue when I try to install from source. I have to copy the generated parser_table to the root directory.

Reply:
Yes, but we should handle that in a separate PR.  It's probably something we missed in the `setup.py` file